### PR TITLE
Integrate test last run summary in Home page

### LINKED ISSUE #1324: Integrate test last run summary in Home page
**Context:** As part of the `transactions` redesign, we want to include extra information in the tests home page. In the screenshot you can see that now we have the `count` of runs, the `last run time` and `last run result`. The information was already included in the response of the `/tests` endpoint [here](https://github.com/kubeshop/tracetest/pull/1318).

Screen Shot 2022-09-28 at 10 19 40

**AC1**
As a user in the Tests Home page,
I want to see the `last run` summary as part of the test card.

This should include:
- Counter of runs
- Last run time
- Last run result

### PR DESCRIPTION
**Pull request by jorgeepc**
**Context:** As part of the `transactions` redesign, we want to include extra information in the tests home page. The information was already included in the response of the `/tests` endpoint [here](https://github.com/kubeshop/tracetest/pull/1318). This PR includes the information in the UI and also updates the feature flag to enable the new design.

## Changes

- Add last run summary in Home page
- Update feature flags to enable the new design

## Fixes

- fixes #1324

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Screenshot

Screen Shot 2022-10-03 at 17 56 36

Screen Shot 2022-10-03 at 17 56 43

### CODE REVIEW

File: web/public/index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
           env: getTemplateValue('{{ .Env }}', ''),
           demoEnabled: getTemplateValue('{{ .DemoEnabled }}', '["pokeshop", "otel"]'),
           demoEndpoints: getTemplateValue('{{ .DemoEndpoints }}', '{"PokeshopHttp": "http://demo-pokemon-api.demo.svc.cluster.local", "PokeshopGrpc": "demo-pokemon-api.demo.svc.cluster.local:8082", "OtelFrontend": "http://otel-frontend:8084" }'),
-          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '[]'),
+          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '["transactions1"]'),
```
Comment (schoren) [inline web/public/index.html:48]: is that `1` at the end of the default value (`transactions1`) ok?
Also, I would expect the default to be "no experiments enabled"

**Review by schoren — APPROVED** [review]
LGTM!

File: web/public/index.html (hunk 1 above)
Comment (xoscar) [inline web/public/index.html:48]: Do we want to keep it enabled by default now?

File: web/src/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -23,13 +24,16 @@ const menuItems = [
     label: 'Tests',
     path: '/',
   },
-  {
+];
+
+if (ExperimentalFeature.isEnabled('transactions')) {
```
Comment (xoscar) [inline web/src/components/Layout/Layout.tsx:29]: Good catch!

File: web/src/models/Test.model.ts [hunk 3]
```diff
@@ -9,13 +9,15 @@ const Test = ({
...
   serviceUnderTest: rawTrigger,
+  summary = {},
 }: TRawTest): TTest => ({
   id,
   name,
   version,
   description,
   definition: TestSpecs(specs || {}),
   trigger: Trigger(rawTrigger || {}),
+  summary,
```
Comment (xoscar) [inline web/src/models/Test.model.ts:20]: Do we need to create a model for the new info? or do you think we should be OK with how it is?

File: web/src/pages/Home/TestList.tsx [hunk 4]
```diff
@@ -50,13 +48,9 @@ const TestList = ({query}: IProps) => {
...
     >
       <S.TestListContainer data-cy="test-list">
-        {list?.map(test =>
-          ExperimentalFeature.isEnabled('transactions') ? (
-            <TestCardV2 key={test.id} onDelete={onDelete} onRun={onRunTest} onViewAll={onClick} test={test} />
-          ) : (
-            <TestCard test={test} onClick={onClick} onDelete={onDelete} onRunTest={onRunTest} key={test.id} />
-          )
-        )}
+        {list?.map(test => (
```
Comment (xoscar) [inline web/src/pages/Home/TestList.tsx:51]: 🔥

**Review by xoscar — APPROVED** [review]
GJ @jorgeepc! I left some minor comments

File: web/public/index.html (hunk 1 above)
Comment (cescoferraro) [reply web/public/index.html:48]: I think so!

**Review by cescoferraro — APPROVED** [review]
Great job @jorgeepc

File: web/public/index.html (hunk 1 above)
Comment (jorgeepc) [reply web/public/index.html:48]: This was a mistake @schoren Fixing it right now.

File: web/src/models/Test.model.ts (hunk 3 above)
Comment (jorgeepc) [reply web/src/models/Test.model.ts]: Created a model. Thanks for the suggestion.

File: web/public/index.html (hunk 1 above)
Comment (jorgeepc) [reply web/public/index.html]: This was a mistake. Fixed!

**Pull request merged**